t/plugin/mocking.t

```diff
@@ -424,3 +424,44 @@ passed
 GET /hello
 --- response_body chomp
 empty_var:
+
+
+
+=== TEST 19: set route(return headers)
+--- config
+       location /t {
+           content_by_lua_block {
+               local t = require("lib.test_admin").test
+               local code, body = t('/apisix/admin/routes/1',
+                    ngx.HTTP_PUT,
+                    [[{
+                           "plugins": {
+                               "mocking": {
+                                   "response_example": "hello world",
+                                   "response_headers": {
+                                        "X-Apisix": "is-cool",
+                                        "X-Really":"yes"
```

Comment:
What happens if you set 2 identical response headers?  Add a test case

Reply:
What do you mean by "identical response headers"?

Reply:
```
                                   "response_headers": {
                                        "X-Apisix": "is-cool",
                                        "X-Apisix": "yes"
                                    }
                               }
```

Reply:
1. Only the last one is retained, the first one is lost. However, if someone needs multiple values for a header they can do `"header-name": "val1, val2"`.

2. Why is this test case needed?

Reply:
You could test, I don't think you are right.

Reply:
I said this because you use the function `core.response.add_header` to add the response header, the function will append the value to the same header, but i tested locally, and found that the `conf.response_headers` won't pass the duplicated headers. 
If we pass 
```
                                   "response_headers": {
                                        "X-Apisix": "is-cool",
                                        "X-Apisix": "yes"
                                    }
```
we will get
```
                                   "response_headers": {
                                        "X-Apisix": "yes"
                                    }
```
So it's ok for this.
But i don't see you example